src/patternfly/components/Popover/examples/popover-bottom-example.hbs

```diff
@@ -1,9 +1,9 @@
 {{#> popover popover--modifier="pf-m-bottom" popover--attribute='aria-labelledby="popover-bottom-header" aria-describedby="popover-bottom-body"'}}
   {{#> popover-content}}
     {{#> popover-header}}
```

Comment:
I wonder if we need this element? From reading the [header w3 spec](https://www.w3.org/TR/html5/sections.html#the-header-element) I don't think this is necessary. I just dropped a heading element (pf-c-title) in the empty state component. I bet we could structure this similarly - https://pf-next.com/components/EmptyState/examples/.

Reply:
If we do drop this element, we should add `.pf-c-title` to the usage table.

Reply:
oh good call! yeah, let's do that.

Reply:
Do we need to add a footer to the popover? (where "Links can be used" is in the design)

Reply:
yes good point! i will add that point in, similar to modals